domains/clarenisme.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "razelleclaren",
+    "email": "gracela.claren1@gmail.com"
+  },
+  "record": {
+    "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
+  }
```

Comment:
replace the unknown space-like characters with actual spaces

one before `~` and 2 before `}`